packages/Meshfree/src/DTK_DetailsSVDImpl.hpp

```diff
@@ -240,9 +238,9 @@ struct SVDFunctor
             }
 
             // Obtain left and right Givens rotations by using 2x2 SVD
-            matrix_2x2_type Apq = {{{{E( p, p ), E( p, q )}},
-                                    {{E( q, p ), E( q, q )}}}},
-                            L, D, R;
+            matrix_2x2_type Apq = {
+                {{{E( p, p ), E( p, q )}}, {{E( q, p ), E( q, q )}}}};
+            matrix_2x2_type L, D, R;
```

Comment:
[ES.10: Declare one name (only) per declaration](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-name-one)

Reply:
Sounds like overkill to me. I see rationale in some cases, though.

Reply:
i will admit i really don't like multiple declarations either and try not do do it - independent of whether or not constructors are called.